wgsl/index.bs

```diff
@@ -6668,6 +6668,109 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
     | [=syntax/bitwise_expression=]
 </div>
 
+## Operator Precedence and Associativity ## {#operator-precedence-associativity}
+
+This entire subsection is non-normative.
+
+Operator precedence and associativity in right-hand side WGSL expressions
+emerge from their grammar in summary. Right-hand expressions group operators to organize them,
+as illustrated by the following diagram:
+
+<div class=operators1>
+  <figure>
+    <figcaption>Operator precedence and associativity graph</figcaption>
+    <object type="image/svg+xml" data="img/operators_1.mmd.svg"></object>
+  </figure>
+</div>
+
+To promote readability through verbosity, the following groups do not associate with other groups:
+
+* Short-circuit OR (can associate with self and relational weakly),
+* Short-circuit AND (can associate with self and relational weakly),
+* Binary OR (can associate with self and unary weakly),
+* Binary AND (can associate with self and unary weakly),
+* Binary XOR (can associate with self and unary weakly).
```

Comment:
Links to the tokens/grammar would be beneficial here.

Reply:
I agree, sorry for the overlook! Tried to fix!